Formula/b/basti.rb

```diff
@@ -0,0 +1,31 @@
+require "language/node"
+
+class Basti < Formula
+  desc "Cloud credentials manager cli"
+  homepage "https://www.basti.app/"
+  url "https://registry.npmjs.org/basti/-/basti-1.6.2.tgz"
+  sha256 "8d813c1f4e3b8195655d40e670aa8a2eb7dce2cd21d996564c56a3296163f1d7"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+
+    # Remove incompatible pre-built binaries
+    os = OS.kernel_name.downcase
+    arch = Hardware::CPU.intel? ? "x64" : Hardware::CPU.arch.to_s
+    node_modules = libexec/"lib/node_modules/basti/node_modules"
+    rm_f node_modules/"basti-session-manager-binary-#{os}-#{arch}/session-manager-plugin"
```

Comment:
Since this is removed, should we add a caveat to install the cask `session-manager-plugin` or somehow allow the cask to be a dependency of this formula ?

Reply:
it should only remove the prebuilt binary which does not match with the architecture.